### PR TITLE
Add connection to running interactive job

### PR DESCRIPTION
**Pull request by vlk-jan**
It allows users to connect to running interactive SLURM jobs.

### CODE REVIEW

File: commands/rci/connect-job [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+echo ""
+echo "Jobs for user $USER:"
+
+# Print header with no line number
+echo "$user_jobs" | awk 'NR == 1 {print $0}'
+echo "$user_jobs" | awk 'NR > 1 {print "["NR-1"]", $0}'
+echo ""
+
+# Prompt the user to select a job
+read -r -p "Enter the number of the job to cancel: " job_number
```
Comment (aleskucera) [inline commands/rci/connect-job]: Please rephrase the prompt to make sense.

File: commands/rci/connect-job [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Extract the job ID corresponding to the selected number
+job_id=$(echo "$user_jobs" | awk 'NR > 1' | awk -v job_number="$job_number" 'NR == job_number {print $1}')
+
+# Check if a valid job ID was obtained
+if [ -z "$job_id" ]; then
+    echo "Invalid job number."
+    exit 1
+fi
+
+# Cancel the selected job
```
Comment (aleskucera) [inline commands/rci/connect-job]: This comment as well.

**Review by aleskucera — CHANGES_REQUESTED** [review]
There are artifacts from copying the cancel-job file. Please rephrase the prompts and comments of the `connect-job` script to match the functionality.

**Comment by vlk-jan** [comment]
Text changes completed. I also added the script description to README.

**Comment by vlk-jan** [comment]
Updated README to reflect the current status of the repository.

**Review by aleskucera — APPROVED** [review]

**Pull request merged**